docs/iov-name-service/validator/01-setup.md

```diff
@@ -1,21 +1,16 @@
 ---
 id: setup
-title: Validator setup
-sidebar_label: Setup
+title: Setup a Validator Node
+sidebar_label: Setup a Validator Node
```

Comment:
I chose single nouns for the side_bar label and would like to maintain the parallelism with them.  Swithcing from nouns to phrases breaks the parallelism.

Reply:
what is the rationnal of using nouns only? Cosmos SDK Documentation use sometimes verbs and nouns